src/cf-core/usage.md

```diff
@@ -1044,164 +1041,6 @@ _Note that it is best practice to document the URL of a quoted work using the ci
 </blockquote>
 ```
 
-
-## Base forms
-
-_Visit https://github.com/cfpb/cf-forms for advanced form field patterns._
-
-### Form labels
-
-#### Default label
-
-<label>Form label</label>
-
-```
-<label>Form label</label>
-```
-
-#### Label for a radio or checkbox
-
-<input type="radio" id="radio-input">
-<label for="radio-input">Radio label</label>
-<input type="checkbox" id="checkbox-input">
-<label for="checkbox-input">Checkbox label</label>
-
-```
-<input type="radio" id="radio-input">
-<label for="radio-input">Radio label</label>
-<input type="checkbox" id="checkbox-input">
-<label for="checkbox-input">Checkbox label</label>
-```
-
-### Form elements
-
-_Note that the `.focus` class is for documentation purposes only and should not be used in production._
-
-#### Text input
-
-<input type="text" value="Lorem ipsum">
-<input class="focus" type="text" value="Lorem ipsum">
-<br><br>
-<input placeholder="Lorem ipsum" type="text" value="">
-
-```
-<input type="text" value="Lorem ipsum">
-<input class="focus" type="text" value="Lorem ipsum">
-<br><br>
-<input placeholder="Lorem ipsum" type="text" value="">
-```
-
-#### Search input
-
-<input type="search" value="Lorem ipsum">
-<input class="focus" type="search" value="Lorem ipsum">
-<br><br>
-<input placeholder="Lorem ipsum" type="search" value="">
-
-```
-<input type="search" value="Lorem ipsum">
-<input class="focus" type="search" value="Lorem ipsum">
-<br><br>
-<input placeholder="Lorem ipsum" type="search" value="">
-```
-
-#### Email input
-
-<input type="email" value="Lorem ipsum">
-<input class="focus" type="email" value="Lorem ipsum">
-<br><br>
-<input placeholder="Lorem ipsum" type="email" value="">
-
-```
-<input type="email" value="Lorem ipsum">
-<input class="focus" type="email" value="Lorem ipsum">
-<br><br>
-<input placeholder="Lorem ipsum" type="email" value="">
-```
-
-#### URL input
-
-<input type="url" value="Lorem ipsum">
-<input class="focus" type="url" value="Lorem ipsum">
-<br><br>
-<input placeholder="Lorem ipsum" type="url" value="">
-
-```
-<input type="url" value="Lorem ipsum">
-<input class="focus" type="url" value="Lorem ipsum">
-<br><br>
-<input placeholder="Lorem ipsum" type="url" value="">
-```
-
-#### TEL input
-
-<input type="tel" value="Lorem ipsum">
-<input class="focus" type="tel" value="Lorem ipsum">
-<br><br>
-<input placeholder="Lorem ipsum" type="tel" value="">
-
-```
-<input type="tel" value="Lorem ipsum">
-<input class="focus" type="tel" value="Lorem ipsum">
-<br><br>
-<input placeholder="Lorem ipsum" type="tel" value="">
-```
-
-#### Number input
-
-<input type="number" value="1000">
-<input class="focus" type="number" value="1000">
-<br><br>
-<input placeholder="Lorem ipsum" type="number" value="">
-
-```
-<input type="number" value="1000">
-<input class="focus" type="number" value="1000">
-<br><br>
-<input placeholder="Lorem ipsum" type="number" value="">
-```
-
-#### Textarea
```

Comment:
I think it's just in cfgov-refresh right now. I did a search of all the less files in the project and I don't see any results. Should this go on the D+R backlog or the issues list here to move over those styles?